IPython/lib/deepreload.py

```diff
@@ -320,13 +319,10 @@ def deep_reload_hook(m):
     return newm
 
 # Save the original hooks
-try:
-    original_reload = builtin_mod.reload
-except AttributeError:
-    original_reload = imp.reload    # Python 3
+original_reload = imp.reload    # Python 3
```

Comment:
Ah, rats, we still need `imp.reload` while we're supporting 3.3. I'll drop the last commit.